canonical-websites/www.ubuntu.com
# 1649 homepage

## Done

* converted homepage to vbt and the new design
* updated Landscape logo

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/))
- compare to the [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit) and the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/Homepage/Deliverables?preview=homepage+v5+mix.png)

## Issue / Card

Fixes #1649 
Fixes #1567
